src/ExpressiveInstaller/OptionalPackages.php

```diff
@@ -42,27 +42,27 @@ class OptionalPackages
     /**
      * @var array
      */
-    private static $config;
+    public static $config;
```

Comment:
Couldn't these assignments be done with the Reflection API instead? That way we protect encapsulation for normal consumption.